include/effects/matrix/PatternWeather.h

```diff
@@ -227,10 +255,9 @@ class PatternWeather : public LEDStripEffect
                     if (main["temp_min"] > 0)
                         lowTemp         = KelvinToLocal(main["temp_min"]);
 
-                    String iconIdTomorrow = entry["weather"][0]["icon"];
-                    iconTomorrow = iconIdTomorrow.toInt();
+                    iconTomorrow = String(entry["weather"][0]["icon"]);

```

Comment:
The explicit construction of the String is not necessary; ArduinoJson is pretty clever when it comes to figuring out what C++ type/class to convert to.

Reply:
That's what I thought too, but I get this error without it.
include/effects/matrix/PatternWeather.h:258:62: error: ambiguous overload for 'operator=' 
Turns out it is my lack of familiarity with the ArduinoJson library. turns out you just need to use .as<String>() to do it directly with the library.